integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -65,7 +63,9 @@ run_sonobuoy() {
 
 	# Uncompress results
 	ls | grep tar.gz | xargs tar -xvf
-	e2e_result_log="${e2e_result_dir}/plugins/e2e/results/e2e.log"
+
+	# e2e results log will be on "plugins/e2e" directory
+	e2e_result_log=$(find ./plugins/e2e -name "e2e.log")
```

Comment:
what happen if there are more than one `e2e.log` file?

Reply:
this is executed on a temp directory, which changes every time it is executed, so there shouldn't be more than one

Reply:
thanks

Reply:
using `find` feels... odd - presumably the log file moved path recently, and might do so again. So, sure, it's a fix - let's see if it 'holds water'...